src/sentry/static/sentry/app/views/projectDetail/charts/projectStabilityChart.tsx

```diff
@@ -161,8 +180,24 @@ class Chart extends React.Component<ChartProps> {
     return false;
   }
 
+  // inspired by app/components/charts/eventsChart.tsx@handleLegendSelectChanged
+  handleLegendSelectChanged = legendChange => {
```

Comment:
can we add a type here?

Reply:
had to do some digging in echarts but should be good now 👍